src/main/java/frc/team2412/robot/subsystems/ShooterSubsystem.java

```diff
@@ -48,14 +56,25 @@
     @Log
     public final Servo servo2;
 
+    public final CANPIDController leftPIDController;
+    public final CANPIDController rightPIDController;
+
 	// Private so other subsystems dont mess with it but this one can
 	private ShooterSubsystemState shooterSubsystemState;
 
 	// Create the example subsystem
-	public ShooterSubsystem(CANSparkMax motor1, CANSparkMax motor2, CANSparkMax motor3, Servo servo1, Servo servo2) {
-		this.motor1 = motor1;
-        this.motor2 = motor2;
+	public ShooterSubsystem(CANSparkMax leftMotor, CANSparkMax rightMotor, CANSparkMax motor3, Servo servo1, Servo servo2) {
+        this.leftMotor = leftMotor;
+        this.leftPIDController = this.leftMotor.getPIDController();
+
+        this.rightMotor = rightMotor;
+        this.rightMotor.setInverted(true);
```

Comment:
Best to also set the motors to coast mode, else they're likely going to be damaged or draw a lot of current when they are set to stop after running at high speed.